kanidmd_web_ui/src/views/profile.rs

```diff
@@ -1,3 +1,4 @@
+use crate::constants::CSS_PAGE_HEADER;
```

Comment:
We could also consider importing and allowing a customised css file too somewhere? But I'm not smart enough at web things for that.

Reply:
That's possible, and I can imagine some branding/UI customisation will come further down the track, but these constants are the classes that get applied to individual UI elements, so it'd still need to be defined somewhere.

Reply:
yeah, zero pressure now :)